app/src/main/java/com/mapbox/maps/testapp/examples/ScaleBarActivity.kt

```diff
@@ -0,0 +1,16 @@
+package com.mapbox.maps.testapp.examples
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.testapp.R
+
+/**
+ * Activity to showcase scale bar custom configuration using xml attributes.
+ */
+class ScaleBarActivity : AppCompatActivity() {
```

Comment:
should we create similar activity on iOS for parity and making sure that our scalebar customizations are aligned? @mapbox/maps-ios